zuul.d/projects.yaml

```diff
@@ -4,6 +4,12 @@
       jobs:
         - openlab-zuul-jobs-check
 
+- project:
+    name: theopenlab/labkeeper
```

Comment:
Good idea, thank you @liu-sheng 
@Yikun , how about adding a periodic job to execute labchecker at here in separated PR? just checking and reporting, operator can recover later if issue exists

Reply:
I was consider add a periodic job to check, but it seems this job it's a little bit special (if the job means creating new vm + adding openstack provider config + doing the remote cloud provider check):
- This job should have the `cloud.yaml` info( for now we need user type `vault-password.txt` and then decrypt the template file to generate the `cloud.yaml`.
- This job can **only** check the cloud provider related testcase, the check item is limited.

But the alternative way is that:
1. Set the nodepool vm as [static pool](https://zuul-ci.org/docs/zuul/admin/nodepool_static.html), named as ` nodepool-allinone`
2. Add a check job, nodesets is set ` nodepool-allinone`, then the job will only scheduled to the real nodepool node, and this job will check the real nodepool or zuul env.
3. the failed check will be reported to the issue page of OpenLab.

If we do this periodic like above, we will use the `cloud.yaml` in real env, and all items would be checked in real env.

Reply:
After F2F discussion:
1. We can add the labchecker call after deploy.
2. We can add the periodic check using crontab.
3. We can add the check to ansible role.

Reply:
@Yikun , exactly, thank you